src/actions/requests.rs

```diff
@@ -83,6 +84,9 @@ impl RequestAction for WorkspaceSymbol {
 
         Ok(
             defs.into_iter()
+                // Sometimes analysis will return duplicate symbols
+                // for the same location, fix that up.
+                .unique_by(|d| d.span.clone())
```

Comment:
Atom doesn't support _workspace/symbol_ yet so I haven't used this myself, and I'm not sure if it's possible to get a different named symbol in the same location. You mentioned `include!` as possible causing this?

But if so perhaps we should include the `d.name` in the unique check too.

Reply:
> But if so perhaps we should include the d.name in the unique check too.


Seems reasonable, yeah! I think in theory macros can generate stuff with same spans.